containers/azure-functions-pwsh-6.2.3/README.md

```diff
@@ -0,0 +1,59 @@
+# Azure Functions & Pwsh (.NET Core Latest)
+
+## Summary
+
+*Develop Azure Functions in PowerShell. Includes NET Core (Latest), the Azure Functions SDK, and related extensions and dependencies.*
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | The VS Code Team |
+| *Definition type* | Dockerfile |
+| *Languages, platforms* | Azure Functions, .NET Core, PowerShell |
+
+## Using this definition with an existing folder
+
+This definition requires an Azure subscription to use. You can create a [free account here](https://azure.microsoft.com/en-us/free/serverless/) and learn more about using [Azure Functions with VS Code here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-function-vs-code). Once you have an Azure account, follow these steps:
+
+1. If this is your first time using a development container, please follow the [getting started steps](https://aka.ms/vscode-remote/containers/getting-started) to set up your machine.
+
+2. To use VS Code's copy of this definition:
+   1. Start VS Code and open your project folder.
+   2. Press <kbd>F1</kbd> select and **Remote-Containers: Add Development Container Configuration Files...** from the command palette.
+   3. Select the Azure Functions & pwsh (.NET Core 2.2) definition.
+
+3. To use latest-and-greatest copy of this definition from the repository:
+   1. Clone this repository.
+   2. Copy the contents of `containers/azure-functions-dotnetcore-latest/.devcontainer` to the root of your project folder.
```

Comment:
this says dotnetcore

Reply:
Changed to `containers/azure-functions-pwsh-6/.devcontainer`